revolv/lib/tests/test_mailer.py

```diff
@@ -62,3 +68,77 @@ def test_cc_admins(self):
         self.assertEqual(len(mail.outbox), 1)
         self.assertIn("a@a.com", mail.outbox[0].cc)
         self.assertIn("b@b.com", mail.outbox[0].cc)
+
+
+class MonthlyDonationEmailTestCase(TestCase):
+    def test_monthly_donation_email_no_donations(self):
+        """
+        Tests that if there are no donations, nothing will be sent
+        """
+        RevolvUserProfile.factories.base.create()
+        call_command('monthlydonationemail', override=True)
+        self.assertEqual(len(mail.outbox), 0)
+
+    def test_monthly_donation_email(self):
+        """
+        Tests that with two donations in the past month from two different users,
+        the Revolv Mailer send two emails
+        """
+        # creates 2 users and 2 projects
+        user1, user2 = RevolvUserProfile.factories.base.create_batch(2)
+        project1, project2 = Project.factories.base.create_batch(2)
+
+        # gets a day from the last month
+        date_of_last_month = datetime.date.today().replace(day=1) - datetime.timedelta(days=1)
+
+        # creates two payments and sets their created_at date to a date from last month
+        Payment.factories.base.create(
```

Comment:
You actually don't need to specify all of these values, you can just say `Payment.factories.donation.create(created_at=...)`. Everything else doesn't really matter, so no need to include it